hyp3_sdk/hyp3.py

```diff
@@ -219,47 +219,120 @@ def prepare_autorift_job(cls, granule1: str, granule2: str, name: Optional[str]
             job_dict['name'] = name
         return job_dict
 
-    def submit_rtc_job(self, granule: str, name: Optional[str] = None, **kwargs) -> Batch:
+    def submit_rtc_job(self,
+                       granule: str,
+                       name: Optional[str] = None,
+                       dem_matching: Optional[bool] = None,
+                       include_dem: Optional[bool] = None,
+                       include_inc_map: Optional[bool] = None,
+                       include_scattering_area: Optional[bool] = None,
+                       radiometry: Union[Literal['gamma0'], Literal['sigma0']] = None,
```

Comment:
I think you can just do

`radiometry: Literal['gamma0', 'sigma0'] = None,`

https://docs.python.org/3/library/typing.html#typing.Literal